central/runtimeconfiguration/store/postgres/gen.go

```diff
@@ -0,0 +1,3 @@
+package postgres
+
+//go:generate pg-table-bindings-wrapper --type=storage.RuntimeFilterData
```

Comment:
The name of the table will likely be changed to collector_runtime_config or something similar.